portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,83 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+/* This code script executes when the website is first loaded in  */
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+/* JavaScript "clicks" on the default tab so that content is shown when first going into the website */
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+
+/*global variable that gives me which slide I'm currently on in the SlideShow*/
+var slideIndex;
+
+/* Performs necessary functions when page is first loaded */
```

Comment:
Here and others, use jsdoc for methods: https://google.github.io/styleguide/jsguide.html#jsdoc